pom.xml

```diff
@@ -126,6 +126,11 @@
             <artifactId>jaxb-runtime</artifactId>
             <version>2.4.0-b180725.0644</version>
         </dependency>
+        <dependency>
+            <groupId>io.airlift</groupId>
+            <artifactId>airline</artifactId>
+            <version>0.8</version>
+        </dependency>
```

Comment:
please check the license of this dependency, and update the LICENSE and NOTICE file

Reply:
Actually, it has ASF license.

Reply:
"Airlift is a framework for building REST services in Java." 
Ah? did you add a REST service??
by the way, we need to add comments when we add new dependencies.

Reply:
No， I only use its airline module.

Reply:
Is this library's function similar to 
```
<dependency>
           <groupId>commons-cli</groupId>
            <artifactId>commons-cli</artifactId>
            <version>${common.cli.version}</version>
 </dependency>
```
in iotdb-cli module ?
Should we choose one and keep our dependencies as less as possible?

Reply:
I have seen the implementation in cli module, and I think this package is more useful and extensible; one for IoTDB module, one for Cli module, it should be fine.